sympy/polys/euclidtools.py

```diff
@@ -1650,7 +1650,11 @@ def dup_rr_lcm(f, g, K):
     h = dup_quo(dup_mul(f, g, K),
                 dup_gcd(f, g, K), K)
 
-    return dup_mul_ground(h, c, K)
+    h = dup_mul_ground(h, c, K)
+
+    u = K.canonical_unit(dup_LC(h, K))
+
+    return dup_mul_ground(h, u, K)
```

Comment:
It would be more efficient to multiply the scalars `u` and `c` once and use only a single call to `dup_mul_ground`.